app/src/main/res/values-es/strings.xml

```diff
@@ -391,6 +391,17 @@
     <string name="settingsAiEngineDescription">Seleccione su asistente de redacción</string>
     <string name="settingsAppLock">Bloqueo de aplicaciones</string>
     <string name="settingsAuthorizeTracking">Autorizar el seguimiento</string>
+    <string name="settingsAutoAdvanceDescription">Seleccione qué mostrar cuando se archiva o elimina un mensaje</string>
+    <string name="settingsAutoAdvanceFollowingThreadDescription">Mostrar la siguiente conversación al archivar o borrar</string>
+    <string name="settingsAutoAdvanceFollowingThreadTitle">Siguiendo la conversación</string>
+    <string name="settingsAutoAdvanceListOfThreadsDescription">Mostrar lista de conversaciones al archivar o borrar</string>
```

Comment:
It's `Mostrar la lista...` you're missing a determinant. You have the same issue with the english translation

Reply:
You still have the issue with the english translation